src/main/java/org/frankframework/frankdoc/model/FrankElement.java

```diff
@@ -164,7 +164,7 @@ private void handlePossibleParameters(FrankClass clazz) {
 		this.meaningOfParameters = clazz.getJavaDocTag(JAVADOC_PARAMETERS);
 		for(String specificParameterStr: clazz.getAllJavaDocTagsOf(JAVADOC_PARAMETER)) {
 			if(StringUtils.isBlank(specificParameterStr)) {
-				log.warn("FrankElement [{}] has specific parameters without a name or description", fullName);
+				log.error("FrankElement [{}] has specific parameters without a name or description", fullName);
```

Comment:
Build moet niet falen als alleen description ontbreekt

Reply:
Als de naam en de beschrijving allebei ontbreken, moet de build dan wel falen?

Reply:
Ja de naam is wel verplicht.

Reply:
Ik heb nu een error als er geen naam en geen description is. Ik heb een warning als er alleen een naam is.